README.md

```diff
@@ -8,14 +8,14 @@ Billy was born as part of [go-git/go-git](https://github.com/go-git/go-git) proj
 ## Installation
 
 ```go
-import "github.com/go-git/go-billy/v5" // with go modules enabled (GO111MODULE=on or outside GOPATH)
-import "github.com/go-git/go-billy" // with go modules disabled
+import "github.com/fluxcd/go-billy/v5" // with go modules enabled (GO111MODULE=on or outside GOPATH)
```

Comment:
What is the reasoning behind changing the module name? Given our goals are not to create a hard fork, I would say this is a conflicting choice (and signals something different). If the argument is versioning, I think Go Modules `replace` works just fine with tags (`replace github.com/go-git/go-billy/v5 => github.com/fluxcd/go-billy/v5 <tag on fork>`).

Reply:
It was mostly to align with the go-git approach. I have now reverted this change.

Reply:
Yes, I was too late with jumping in on that change :grimacing: